core/src/main/java/io/aiven/kafka/tieredstorage/config/RemoteStorageManagerConfig.java

```diff
@@ -95,6 +97,11 @@ public class RemoteStorageManagerConfig extends AbstractConfig {
         + "Remote Log Segment metadata on Remote Log Metadata Manager back-end. "
         + "Allowed values: " + Arrays.toString(SegmentCustomMetadataField.names());
 
+    private static final String UPLOAD_RATE_LIMIT_BYTES_CONFIG = "upload.rate.limit.bytes.per.second";
```

Comment:
Kafka itself uses the wording "byte rate" instead of "bytes per second" in configs (e.g. quotas). Shall we too?
```suggestion
    private static final String UPLOAD_RATE_LIMIT_BYTES_CONFIG = "upload.byte.rate.limit";
```

Reply:
OTOH, it's not that important and I don't know if it's really better

Reply:
Great, let's do this.